apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -97,58 +97,26 @@ public abstract partial class BigtableClient
         /// "default" application profile will be used.
         /// </param>
         /// <returns>The created <see cref="BigtableClient"/>.</returns>
-        public static BigtableClient Create(BigtableServiceApiSettings settings = null, ClientCreationSettings clientCreationSettings = null, string appProfileId = null)
-        {
-            clientCreationSettings?.Validate();
-            var channelCredentials = clientCreationSettings?.Credentials;
-
-            // Use default credentials if none given.
-            if (channelCredentials == null)
-            {
-                var credentials = GoogleCredential.GetApplicationDefault();
-                if (credentials.IsCreateScopedRequired)
-                {
-                    credentials = credentials.CreateScoped(BigtableServiceApiClient.DefaultScopes);
-                }
-                channelCredentials = credentials.ToChannelCredentials();
-            }
-            var clientCount = clientCreationSettings?.ClientCount ?? Environment.ProcessorCount;
-
-            var endpoint = clientCreationSettings?.ServiceEndpoint ?? BigtableServiceApiClient.DefaultEndpoint;
-            var clients = new BigtableServiceApiClient[clientCount];
-            var shutdowns = new Func<Task>[clientCount];
-            // Fill clients[] with BigtableServiceApiClient instances, each with specific channel
-            for (int i = 0; i < clientCount; i++)
-            {
-                var channel = new Channel(endpoint.Host, endpoint.Port, channelCredentials);
-                clients[i] = BigtableServiceApiClient.Create(channel, clientCreationSettings?.BigtableServiceApiSettings);
-                shutdowns[i] = channel.ShutdownAsync;
-            }
-            Func<Task> shutdown = () => Task.WhenAll(shutdowns.Select(x => x()));
-
-            return new BigtableClientImpl(appProfileId, clients) { UnderlyingClientSettings = settings };
-        }
+        public static BigtableClient Create(ClientCreationSettings clientCreationSettings = null, string appProfileId = null) =>
+            Task.Run(() => CreateAsync(clientCreationSettings, appProfileId)).ResultWithUnwrappedExceptions();
 
         /// <summary>
-        /// Creates a <see cref="BigtableClient"/> which uses the specified channel for remote operations.
+        /// Synchronously creates a <see cref="BigtableClient"/>, applying defaults for all unspecified settings,
+        /// and creating a channel connecting to the given endpoint with application default credentials where
+        /// necessary.
```

Comment:
This comment does not apply here: no settings are specified and no channels get created. It needs to be updated for this overload.